### PR TITLE
Fix json ordering

### PR DESCRIPTION
**Pull request by cosunae**
regenerate references for https://github.com/MeteoSwiss-APN/dawn/pull/184

### CODE REVIEW

**Comment by cosunae** [comment]
launch pipeline

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:cosunae/dawn.git"
```
Comment (twicki) [inline bundle/CMakeLists.txt]: remember to change this back!

File: test/integration-test/PassStageMerger/Test01_after_ref.json [hunk 2]
```diff
@@ -1,145 +1,133 @@
...
-          "name": "field_b0"
+          }
         },
-        {
+        "field_a1": {
           "IsTemporary": false,
           "dim": "[1,1,1]",
           "field": {
-            "accessID": 18,
+            "accessID": 16,
```
Comment (twicki) [inline test/integration-test/PassStageMerger/Test01_after_ref.json]: it seems that from testing some ID's are not generated correctly

Comment (cosunae) [reply test/integration-test/PassStageMerger/Test01_after_ref.json]: what do you mean, this is just the reordering, that we now fixed (and is different than the ref in master)

**Comment by cosunae** [comment]
launch pipeline

**Comment by cosunae** [comment]
launch pipeline

File: bundle/CMakeLists.txt (hunk 1 above)
Comment (cosunae) [inline bundle/CMakeLists.txt]:
```suggestion
    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
```

File: bundle/CMakeLists.txt [hunk 3]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:cosunae/dawn.git"
+    GIT_TAG "fix_json_ordering"
```
Comment (cosunae) [inline bundle/CMakeLists.txt]:
```suggestion
    GIT_TAG "master" 
```

**Pull request merged**